tensorly/decomposition/_cp.py

```diff
@@ -446,6 +446,10 @@ def sample_khatri_rao(matrices, n_samples, skip_matrix=None,
     skip_matrix : None or int, optional, default is None
         if not None, index of a matrix to skip
 
+    indices_list : list, default is None
```

Comment:
```suggestion
    indices_list : list, default is None
        Contains, for each matrix in `matrices`, a list of indices of rows to sample.
        if None, random indices will be created
```
Slight suggestion for clarity